mne/encoding/model.py

```diff
@@ -0,0 +1,323 @@
+"""Base and high-level classes for fitting encoding models."""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#          Jona Sassenhagen <jona.sassenhagen@gmail.de>
+#          Jean-Remi King <jeanremi.king@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from .feature import EventsBinarizer, DataDelayer
+from ..io.pick import pick_types, pick_info
+from ..externals.six import string_types
+from ..evoked import EvokedArray
+
+
+class EventRelatedRegressor(object):
```

Comment:
Maybe it would make sense to leave out event-based predictors for now and factor them in later?..